Lombiq.Tests.UI/Services/WebDriverFactory.cs

```diff
@@ -48,6 +48,7 @@ ChromeDriver CreateDriverInner(ChromeDriverService service)
             configuration.BrowserOptionsConfigurator?.Invoke(chromeConfig.Options);
 
             chromeConfig.Service = service ?? ChromeDriverService.CreateDefaultService();
+            chromeConfig.Service.SuppressInitialDiagnosticInformation = true;
```

Comment:
Isn't this available for the other drivers?

Reply:
I've found it for Edge, but not for Fx or IE.